diff --git a/examples/Cargo.toml b/examples/Cargo.toml
index ec493dd..26efff9 100644
--- a/examples/Cargo.toml
+++ b/examples/Cargo.toml
@@ -35,7 +35,7 @@ arbitrary-model-tests = { path = "../lib" }
 better-panic = "0.2"
 honggfuzz = "0.5"
 
-ahash = "0.3"
+ahash = "0.4"
 hashbrown = "0.8"
 indexmap = "1.4"
 linked-hash-map = "0.5"
diff --git a/examples/src/btree_map.rs b/examples/src/btree_map.rs
index 803974f..83743a7 100644
--- a/examples/src/btree_map.rs
+++ b/examples/src/btree_map.rs
@@ -1,7 +1,5 @@
 #![allow(clippy::find_map, clippy::filter_map, clippy::must_use_candidate)]
 
-#[macro_use] extern crate arbitrary;
-
 use arbitrary_model_tests::arbitrary_stateful_operations;
 use honggfuzz::fuzz;
 
diff --git a/examples/src/hash_map.rs b/examples/src/hash_map.rs
index 2b5e1eb..3c0304c 100644
--- a/examples/src/hash_map.rs
+++ b/examples/src/hash_map.rs
@@ -3,8 +3,6 @@
 #![allow(clippy::must_use_candidate)]
 #![feature(shrink_to)]
 
-#[macro_use] extern crate arbitrary;
-
 use arbitrary_model_tests::arbitrary_stateful_operations;
 use honggfuzz::fuzz;
 
diff --git a/examples/src/linked_hash_map.rs b/examples/src/linked_hash_map.rs
index 18a853b..0201fe8 100644
--- a/examples/src/linked_hash_map.rs
+++ b/examples/src/linked_hash_map.rs
@@ -1,8 +1,6 @@
 #![allow(clippy::find_map)]
 #![allow(clippy::must_use_candidate)]
 
-#[macro_use] extern crate arbitrary;
-
 use arbitrary_model_tests::arbitrary_stateful_operations;
 use honggfuzz::fuzz;
 
diff --git a/lib/Cargo.toml b/lib/Cargo.toml
index 9dcf98a..039b55d 100644
--- a/lib/Cargo.toml
+++ b/lib/Cargo.toml
@@ -6,6 +6,7 @@ authors = ["Jakub Wieczorek <jakub.adam.wieczorek@gmail.com>"]
 edition = "2018"
 license = "MIT"
 keywords = ["fuzzing", "test", "honggfuzz"]
+repository = "https://github.com/jakubadamw/arbitrary-model-tests"
 
 [lib]
 proc-macro = true
diff --git a/README.md b/lib/README.md
similarity index 92%
rename from README.md
rename to lib/README.md
index ed36737..61098bf 100644
--- a/README.md
+++ b/lib/README.md
@@ -1,8 +1,6 @@
 # arbitrary-model-tests
 
 ![Build status](https://github.com/jakubadamw/arbitrary-model-tests/workflows/Build/badge.svg)
-[![crates.io](https://img.shields.io/crates/v/arbitrary-model-tests.svg)](https://crates.io/crates/arbitrary-model-tests)
-[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
 This is an attempt at creating a convenient procedural macro to be used for testing stateful models (in particular, various kinds of data structures) against a trivial (but usually very inefficient) implementation that is semantically 100% equivalent to the target implementation but, in contrast, *obviously* correct. The purpose of the macro is to generate the boilerplate code for testing particular operations of the model so that the user-provided definition of the test for a given stateful structure becomes as succinct as possible.
 
